python/pyspark/sql/functions.py

```diff
@@ -3233,6 +3233,58 @@ def pandas_udf(f=None, returnType=None, functionType=None):
        |  1| 21|
        +---+---+
 
+    6. COGROUPED_MAP
+
+       A cogrouped map UDF defines transformation: two `pandas.DataFrame` -> `pandas.DataFrame`
```

Comment:
I think instead of "two `pandas.DataFrame`", better to show "(`pandas.DataFrame`, `pandas.DataFrame`) -> `pandas.DataFrame`"